mods/persistence/modules/projectiles/overrides.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/gun/launcher/syringe/rapid
+	origin_tech = "{'wormholes':1337}"
+
+/obj/item/gun/energy/temperature
+	origin_tech = "{'wormholes':1337}"
+
+/obj/item/grenade/anti_photon
+	origin_tech = "{'wormholes':1337}"
+
+/obj/item/gun/energy/confuseray
```

Comment:
Ditto

Reply:
as discussed on discord, will add variants for these guns in another pr soon